charts/nri-kube-events/README.md

```diff
@@ -1,57 +1,149 @@
+| [global.]fedramp | `false` |  | For the integrations that support it, enables FedRAMP |
+| [global.]lowDataMode | `false` |  | In each integration it has different behaviour but all aims to send less metrics to the backend to try to save costs |
+| [global.]privileged | Depends on the chart[<sup>3</sup>](#values-managed-globally-3) |  | In each integration it has different behaviour but all aims to send less metrics to the backend to try to save costs |
+| [global.]proxy | `""` |  | Configures the integration to send all HTTP/HTTPS request through the proxy in that URL. The URL should have a standard format like `https://user:password@hostname:port` |
+| [global.]nrStaging | `false` |  | Send the metrics to the staging backend. Requires a valid staging license key |
+| [global.]verboseLog | `false` |  | Sets the debug logs to this integration or all integrations if it is set globally |
+
+### Further information
+<a name="values-managed-globally-1"></a>
+#### 1. Merged
+
+Merged means that the values from global are not replaced by the local ones. Think in this example:
+```yaml
+global:
+  labels:
+    global: global
+  hostNetwork: true
+  nodeSelector:
+    global: global
+
+labels:
+  local: local
+nodeSelector:
+  local: local
+hostNetwork: false
 ```
+
+This values will template `hostNetwork` to `false`, a map of labels `{ "global": "global", "local": "local" }` and a `nodeSelector` with
+`{ "local": "local" }`.
+
+This is the rationale behind of this behavior:
+ * `hostNetwork` is templated to `false` because is overriding the value defined globally.
+ * `labels` are merged because the use may want to label all the New Relic pods at once and label other solution pods differently for
+   clarity' sake.
+ * `nodeSelector` does not merge as `labels` because could make harder to overwrite/delete a selector that comes from global because
+   of the logic that Helm follows merging maps.
+
+It could be confusing to have two behaviours regarding when this chart merges or replaces the `values` from global to local. All functions
+simply replace the value from `global` with the one that is in the root of the chart except functions that template `labels`,
+`customAttributes` and service account's annotations.
+
+<a name="values-managed-globally-2"></a>
+#### 2. Fine grain registries
+
+There are charts that only have 1 image and others that can have 2 or more images. The local path for the registry can change depending
+on the chart itself. If you have an advanced configuration that has a proxy cache for the docker registry you should take a look to the
+chart's `values.yaml` directly to see where you have to change it if you really need so much granularity that needs a proxy/cache registry
+per integration.
+
+<a name="values-managed-globally-3"></a>
+#### 3. Privileged mode
+
+By default, from the common library, the privileged mode is set to false. But most of the helm charts require this to be true to fetch more
```

Comment:
this option has no effect in this chart, no?

Reply:
Nope. This info is now in the common library's PR.